src/app/services/services.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```

Comment:
Why don't we keep it at the top level then? If it gets cluttered, then that's our indication that we should separate things into domains. I mostly don't like the idea of creating a meta-domain for services.